rs/universal_canister/lib/src/lib.rs

```diff
@@ -7,17 +7,15 @@
 
 pub mod management;
 
-use hex_literal::hex;
 use ic_types::Cycles;
 use universal_canister::Ops;
-/// The binary of the universal canister as compiled from
-/// `rs/universal_canister/impl`.
-///
-/// For steps on how to produce it, please see the README in
-/// `rs/universal_canister`.
-pub const UNIVERSAL_CANISTER_WASM: &[u8] = include_bytes!("universal-canister.wasm.gz");
-pub const UNIVERSAL_CANISTER_WASM_SHA256: [u8; 32] =
-    hex!("9c0b4ed1d729ffdd0e8d194df3be621f58a2fb86e1f1bbf556d89f43637de303");
+
+/// The binary of the universal canister as compiled from `rs/universal_canister/impl`.
+pub const UNIVERSAL_CANISTER_WASM: &[u8] = include_bytes!(env!("UNIVERSAL_CANISTER_WASM_PATH"));
+
+pub fn get_universal_canister_wasm_sha256() -> [u8; 32] {
+    ic_crypto_sha2::Sha256::hash(UNIVERSAL_CANISTER_WASM)
+}
```

Comment:
Did you consider this macro? https://docs.rs/sp-core-hashing-proc-macro/latest/sp_core_hashing_proc_macro/index.html

Reply:
Yes I looked into it but didn't want to add a new dependency and would rather use the crypto from the ic repo instead.